securedrop_client/logic.py

```diff
@@ -222,7 +246,6 @@ def start_message_thread(self):
         """
         if not self.message_thread:
             self.message_thread = QThread()
-            self.message_sync = MessageSync(self.api, self.home, self.proxy)
```

Comment:
this change is probably why you're hitting the behavior where files aren't downloading/decrypting (per your comment in our 1:1 earlier today): by moving the definition of `self.message_sync` to the `Client` initialization, you're using `self.api=None`, so neither the message or reply threads is authenticated